projects/openvswitch/fuzzer.options

```diff
@@ -0,0 +1,3 @@
+[libfuzzer]
```

Comment:
This file needs to be named $FUZZ_TARGET.options for it to be used by anything.
You can handle this in the build.sh if you want this to be used by both.

Reply:
Ack.